pom.xml

```diff
@@ -85,7 +85,7 @@
 
         <maven.plugin.antrun.version>3.0.0</maven.plugin.antrun.version>
         <maven.plugin.compiler.version>3.8.1</maven.plugin.compiler.version>
-        <maven.plugin.scala.version>4.4.0</maven.plugin.scala.version>
+        <maven.plugin.scala.version>4.3.0</maven.plugin.scala.version>
```

Comment:
Need a comment here in case that we bump it up wrong again

Reply:
Updated